CONTRIBUTING.md

```diff
@@ -139,6 +143,83 @@ The description field should
 - be neutral and not use biased additions like ~~leading~~, ~~innovative~~, ~~flexible~~ or buzzwords (~~performance~~, ~~next generation~~)
 - not contain hyperlinks or markup
 
+
+### Compatibility information
+
+FMU exporting and importing tools provide information to their users on how they have validated their FMI support.
+Providing and maintaining this information is in the responsibility of the tool vendors. On the tools page of the FMI Webpage, a link to a repository or webpage of the tool vendor will be provided. Tool vendors are encouraged to clearly mark this webpage as provided by them and not by the Modelica Association or the FMI Project, e.g., by prominently placing their logo on this page.
+
+#### Exporting tools
+
+* List the importing tools (including version information), with which the FMUs from this exporter have been imported and simulated successfully.
+This could be (free or commercial) tools available to the tool vendor of the exporting tool. Or the tests could be performed by third parties (users, tool vendors of importing tools). 
+  * optionally additional information needed to reproduce the results of the importers (e.g., compiler flags)
+* Encouragement to provide example FMUs and any information needed to run this FMU and check expected results
+  * check with one of the available checkers before uploading the FMUs (fmpy, VDM Checker, FMU Compliance Checker for FMI 2.0) --> self-certification
+  * description, including with which version of the tool the FMU was generated
+  * (copy description of files from https://github.com/modelica/fmi-cross-check/blob/master/FMI-CROSS-CHECK-RULES.md#914-files-to-submit: TODO: Define if reference result refers to result of the simulation of the original model in the exporting tool, or to the simulation of the FMU)
+  * recommendation: structure your repository according to FMI versions, FMI interface types (ME, CS, SE), platforms (consider to provide source code FMUs)
+
+
+Tool-specific-repository.org:
+```
+2.0
+..
+3.0
+  ME
+    x86_64-windows
+      BouncingBall
+      CoupledClutches
+        CoupledClutches.fmu 
+        CoupledClutches_in.csv 
+        CoupledClutches_ref.csv 
+        CoupledClutches_ref.opt
+        Readme.txt
+    x86-linux  
+    ..
+    source-code
+  CS
+  SE
```

Comment:
> Don't we need to explain the names and what they mean and what they should contain (its partially done for the csv, but though the .opt). For a new person who hasn't been involved in the cross check it might not be obvious.

see latest commit.